src/_posts/databases/mongodb/2000-01-01-compass.md

```diff
@@ -1,72 +1,146 @@
 ---
 title: Access Your MongoDB Database With Compass
 nav: Compass
-modified_at: 2022-04-28 00:00:00
+modified_at: 2022-05-02 00:00:00
 tags: databases mongodb compass tunnel
 index: 4
 ---
 
-MongoDB Compass is the official GUI to manage MongoDB database. It supports CRUD
-operations and offers nice visualization of your data. It is freely available on
-the [MongoDB website](https://www.mongodb.com/products/compass).
+MongoDB Compass is the official GUI to manage MongoDB database.
+It supports CRUD operations and offers nice visualization of your data.
+It is freely available on the
+[MongoDB website](https://www.mongodb.com/products/compass).
 
-Databases hosted on Scalingo are not by default directly available on the
-Internet. There are a couple of ways to get access to a Scalingo hosted
-database: building an SSH tunnel and making the database available on the
-internet. More information in the [dedicated page]({% post_url
-platform/databases/2000-01-01-access %}).
+Databases hosted on Scalingo are not by default directly available on Internet.
+There are a couple of ways to get access to a Scalingo MongoDB hosted database
+with MongoDB Compass:
+* Connection via an Encrypted Tunnel
+* Connection via the DB Tunnel of our CLI
+* Connection with TLS
```

Comment:
I would keep the link to the dedicated page (`platform/databases/2000-01-01-access`)